recipes/landau/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "landau" %}
+{% set version = "1.2.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/landaupkg-{{ version }}.tar.gz
```

```suggestion
  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}pkg/landaupkg-{{ version }}.tar.gz
```

Pipeline failure is because of the name mismatch here.